PyDSS/get_snapshot_timepoints.py

```diff
@@ -0,0 +1,39 @@
+from PyDSS.utils.simulation_utils import create_loadshape_pmult_dataframe_for_simulation
+from PyDSS.reports.reports import logger
+import pandas as pd
+
+
+def get_snapshot_timepoint(dss, options):
+    pv_systems = dss.PVsystems.AllNames()
+    if pv_systems is None:
+        logger.info("No PVSystems are present")
+    pv_profiles_df = pd.DataFrame()
+    for pv_name in pv_systems:

```

Comment:
This will break if pv_systems is None. Same for loads below. Are you sure it won't return an empty list instead?

Reply:
Oh yeah, it returns an empty list if there are no PV systems. 
Will change lines 8 (and the one for loads)

Reply:
if not pv_systems:
        logger.info("No PVSystems are present")

Would this be a better way to check?